src/overlays/actors/ovl_Mir_Ray/z_mir_ray.h

```diff
@@ -9,35 +9,35 @@ struct MirRay;
 
 // Locations of light beams in sMirRayData
 typedef enum {
-    /* 0 */ MIRRAY_SPIRIT_BOMBCHUIWAROOM_DOWNLIGHT,
-    /* 1 */ MIRRAY_SPIRIT_SUNBLOCKROOM_DOWNLIGHT,
-    /* 2 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_DOWNLIGHT,
-    /* 3 */ MIRRAY_SPIRIT_ARMOSROOM_DOWNLIGHT,
-    /* 4 */ MIRRAY_SPIRIT_TOPROOM_DOWNLIGHT,
-    /* 5 */ MIRRAY_SPIRIT_TOPROOM_CEILINGMIRROR,
-    /* 6 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_COBRA,
-    /* 7 */ MIRRAY_SPIRIT_TOPROOM_COBRA1,
-    /* 8 */ MIRRAY_SPIRIT_TOPROOM_COBRA2,
-    /* 9 */ MIRRAY_GANONSCASTLE_SPIRITTRIAL_DOWNLIGHT,
+    /* 00 */ MIRRAY_SPIRIT_BOMBCHUIWAROOM_DOWNLIGHT,
+    /* 01 */ MIRRAY_SPIRIT_SUNBLOCKROOM_DOWNLIGHT,
+    /* 02 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_DOWNLIGHT,
+    /* 03 */ MIRRAY_SPIRIT_ARMOSROOM_DOWNLIGHT,
+    /* 04 */ MIRRAY_SPIRIT_TOPROOM_DOWNLIGHT,
+    /* 05 */ MIRRAY_SPIRIT_TOPROOM_CEILINGMIRROR,
+    /* 06 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_COBRA,
+    /* 07 */ MIRRAY_SPIRIT_TOPROOM_COBRA1,
+    /* 08 */ MIRRAY_SPIRIT_TOPROOM_COBRA2,
+    /* 09 */ MIRRAY_GANONSCASTLE_SPIRITTRIAL_DOWNLIGHT,
     /* 10 */ MIRRAY_MAX,
```

Comment:
Can't do that, 0-prefixed numbers in C are octal (yes, stupid, but true). Use spaces instead.